user-manual/metadata/METS.rst

```diff
@@ -0,0 +1,182 @@
+
+
+.. _amdsec:
+
+<amdSec>
+^^^^^^^^^
+
+There is one amdSec for each object and wraps PREMIS entities as follows:
+
+* PREMIS Object: techMD (tehnical metadata)
+* PREMIS Event: digiprovMD (digital provenance metadata)
+* PREMIS Agent: digiprovMD (digital provenance metadata)
+* PREMIS Rights: rightsMD (rights metadata)
+
+Each amdSec will include one techMD. The techMD is used to wrap PREMIS Object
+semantic units, such as objectIdentifier, objectCharacteristics and
+originalName. See the example techMD below:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 212-258
+
+Each amdsec will include multiple digiprovMDs for PREMIS events related to the
+PREMIS Object. Below is an example of a PREMIS event for a fixity check:
+
+   .. literalinclude:: scripts/METS.xml
+      :language: xml
+      :lines: 820-854
+
+An amdSec for an original object may also contain one or more rightsMDs. The
+rightsMD may contain a reference to metadata in another file, such as rights.csv
+uploaded with a transfer. Below is an example of a PREMIS Rights statement in
+rightsMD section of the AIP METS file:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 2073-2108
+
+An amdSec will also have one ore more sourceMD sections. This is the only
+non-PREMIS section in an amdSec and no XML schema is used for the contents.
+SourceMD is created from the contents of the bag-info.txt whenever a zipped or
+unzipped bag transfer type is used. It is also created for a disk image transfer
+if the user entered meteadata about the disk image in the Transfer tab.
+
+
+.. _filesec:
+
+<fileSec>
+^^^^^^^^^
+
+There is one fileSec listing all files. The fileSec is organized into two or
+more of the following fileGrps:
+
+* original
+* preservation
+* service
+* submissionDocumentation
+* metadata
+* license
+* text/ocr
+* deleted
+
+These are known as USE attributes and must accompany each fileGrp. The fileGrp
+original is required for all METS files. If the AIP includes normalized files
+then the fileGrp preservation is used. The service fileGrp may be used if the
+AIP contains the subfolder (e.g. as the output of digitization workflows).
+
+Each fileGrp must also have a GROUPID and an ID. The GROUPID is the same UUID as
+the ID. Related files will have the same GROUPID.  Below is an example of the
+original fileGrp section in the AIP METS file:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 8830-8850
+
+
+.. _structmap:
+
+<structMap>
+^^^^^^^^^^^
+
+There will be at least one structMap in the METS. It is labeled "Archivematica
+Default" and lists the directories and files in the objects directory.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 8903-8919
+
+If the *document empty directories* microservice runs, then a second structMap
+is created. It is labeled "Normative Directory Structure." At AIP re-ingest the
+new logical structMap will be parsed to re-create the empty directories.
```

Comment:
This is nice, I'm just wondering about teasing apart logical and physical, so I was thinking of something along the lines of: 
```
is created. It is labeled "Normative Directory Structure." At AIP re-ingest this
new 'logical' structMap will be parsed to re-create the empty directories.

A logical structmap describes a relationship between objects in an Archivematica
AIP that might not be true to their 'physical' layout on disk, as is here where 
empty directories have previously been removed but are still recorded so as to
be recreated in the future.
```
What do you think?

Reply:
Yes, I think this makes the difference even clearer for me!